### ISSUE TITLE
Path management

### ISSUE THREAD

**Issue by FFroehlich** [PEtab-dev/libpetab-python#324]
When generating a `petab.Problem` from a yaml file, all files are automatically prefixed by the location of the yaml file. However when directly passing the corresponding `dict` (potentially after manipulation), it is no longer possible to specify a prefix and the only way for the import to work correctly is to `os.chdir` to the appropriate directory.

Straightforward to add a corresponding command line argument, but I guess it would be generally great to store a prefix path somewhere in the generated `petab.Problem`, as this would make it easier to reference files within the problem (not part of the current spec afaik, but it would be great to have in the future).

Opinions?

**Comment by dweindl** [comment]
> However when directly passing the corresponding `dict` (potentially after manipulation), it is no longer possible to specify a prefix and the only way for the import to work correctly is to `os.chdir` to the appropriate directory.

Right. Adding an argument for that to `petab.Problem.from_yaml` would definitely make sense.


> store a prefix path somewhere in the generated `petab.Problem`

Somewhat related, I was planning to introduce some object representing the problem-yaml and adding that to petab.Problem. A base directory could easily be added there. This would also provide access to file names that are otherwise lost, and could be used (in simple cases) as default arguments to `petab.Problem.to_files`, which is currently pretty inconvenient to use...

**Comment by dweindl** [comment]
Closed by #326.
